addons/crm_iap_lead_website/security/crm_iap_lead_website_security.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+<data noupdate="1">
+
+    <record id="crm_reveal_rule_all_rules" model="ir.rule">
```

Comment:
Should be ir_rule_crm_reveal_rule_all for example (ir_rule record model, then some more details).